cx-portal/src/features/admin/userOwn/mapper.ts

```diff
@@ -51,7 +57,10 @@ export const userDetailsToCards = (
         value: parsedToken ? parsedToken.organisation : 'N/A',
       },
       adminName: { label: 'Admin Name', value: 'N/A' },
-      adminMail: { label: 'Admin E-Mail', value: 'N/A' },
+      adminMail: {
+        label: 'Admin E-Mail',
+        value: user.admin?.length ? getAdminEmails(user.admin) : 'N/A',
```

Comment:
sure this value should be an array and not a string?
Anyway, better replace
`getAdminEmails(user.admin)`
with
`user.admin?.map((data: AdminData) => data.email)`